history/characters/16000_gnome.txt

```diff
@@ -1990,6 +1990,162 @@
 	719.12.7={ death=yes }
 }
 
+# dynasty = 10054 [Springcrank]
+16900={
+	name=Gearspinner
+	dynasty=10054
+	culture=gnome religion=rationalism
```

Comment:
Are these gnomes canon?

Reply:
no, i just spent like 3 hours making them for the mechagon thing and was like, might as well put them in the mod